spec/javascripts/components/story/task/task_spec.js

```diff
@@ -45,4 +46,24 @@ describe('<Task />', () => {
       expect(onToggleCheckedBoxSpy).toHaveBeenCalled();
     });
   });
+
+  describe('when task is read-only', () => {
+    it('does not render a Delete span', () => {
+      const { span } = setup({ disabled: true });
```

Comment:
Which span is that? Can we rename it to something clearer? Maybe it could be renamed to `deleteLink`